app/views/results_entities.php

```diff
@@ -191,6 +198,7 @@
       <div dir='ltr' class='result_meta_link'>
         <a class='source_link' href='{$path_locale2}'>&lt;source&gt;</a>
         {$file_bug}
+        {$edit_link}
```

Comment:
You should use the same order of the other view (source, edit, bug from left to right).